### ISSUE TITLE
UnboundLocalError: local variable 'w_pitcher' referenced before assignment

### ISSUE THREAD

**Issue by misea** [zachpanz88/mlbgame#6]
I think this happens when the winning pitcher is not in the stats. Repro:

```
games = mlbgame.games(2015, 4)
for day in games :
	for g in day:
		print(g.game_id)
```

I'm not a python programmer, but I think line 45 of game.py has W_pitcher instead of w_pitcher

**Comment by zachpanz88** [comment]
I see what you mean. I'm currently at a hackathon but I'll issue a fix soon.

**Comment by zachpanz88** [comment]
Fixed in version [2.0.2](https://github.com/zachpanz88/mlbgame/releases/tag/v2.0.2) with commit a6fc64f5ea6889311ec0e8b5deaa4a88de4caf0e. Please update mlbgame for the fix.
